Cargo.toml

```diff
@@ -21,7 +21,7 @@ publish = false
 
 # See https://github.com/cgwalters/cargo-vendor-filterer
 [package.metadata.vendor-filter]
-platforms = ["x86_64-unknown-linux-gnu"]
+platforms = ["x86_64-unknown-linux-gnu", "s390x-unknown-linux-gnu"]
```

Comment:
Should we just be adding aarch64 and ppc64le here?

Reply:
The real fix will be https://github.com/cgwalters/cargo-vendor-filterer/issues/23

I verified that applying the suggestion of

```
$ cargo vendor-filterer --format=tar.gz target/vendor.tar.gz
$ gunzip target/vendor.tar.gz
$ 93769a132991deb03d245e2f9250d0fefa2093da53ced5a64f43d7d5194f78ab  target/vendor.tar
```
gives me an identical sha256 (which you or anyone else can check too since it's reproducible!) both with and without 

```
diff --git a/Cargo.toml b/Cargo.toml
index 8904862a..265f7945 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -21,7 +21,7 @@ publish = false
 
 # See https://github.com/cgwalters/cargo-vendor-filterer
 [package.metadata.vendor-filter]
-platforms = ["x86_64-unknown-linux-gnu", "s390x-unknown-linux-gnu"]
+platforms = ["x86_64-unknown-linux-gnu", "aarch64-unknown-linux-gnu", "powerpc64le-unknown-linux-gnu", "s390x-unknown-linux-gnu"]
 all-features = true
 exclude-crate-paths = [ { name = "curl-sys", exclude = "curl" },
                         { name = "libz-sys", exclude = "src/zlib" },
```

so...let's get this in to not waste CI.